benchmark/compare.js

```diff
@@ -40,6 +40,12 @@ if (benchmarks.length === 0) {
   return;
 }
 
+const cpuCoreSetting = cli.optional.set.find((s) => s.startsWith('CPUCORE='));
```

Comment:
Can we document this somewhere?

Reply:
@anonrig we can add this in the [CLI usage ](https://github.com/nodejs/node/pull/52253/files#diff-b0bed8b607714c703f4de110a731294b2643bf274a536cc00f50fdd90bf60b00L12)

Reply:
we should add it to somewhere, but this is specific to this benchmark not to the node.js cli

Reply:
updated docs writing-and-running-benchmarks.md